test/connectivity/NetworkController/Mode.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    enum Mode
```

Comment:
Nit: ConnectivityTestMode or something like that..

Reply:
I renamed to NetworkContollerMode